javascript/javascript1/week2/fullname-function.js

```diff
@@ -0,0 +1,17 @@
+function getFullname(firstname ,surname,  gender ='', useFormalName = true ){
+  
+  return firstname === '' || surname === '' ? 'Please provide both a first name and a surname.' :
```

Comment:
Try to read this code in two weeks/a month and check if you can actually understand what it does :) 
As much as it is a nice excercise to use the ternary operator, this is an absolute nightmare to read and an instand headache :) Please avoid nesting more then two ternary operators in the future. Preferably use if statements with a return keyword following
```
if(someCondition) {
return someValue;
}
if(someOtherCondition) {
return someOtherValue;
}
//default case
return defaultValue;
```
Readability should always be a top priority when you write code, cause at work you will be returning to what you wrote multiple times and other people will also read and work on it too, so it must be understandable and not give anyone a headache 😉